todo_app/data/mongodb_items.py

```diff
@@ -0,0 +1,61 @@
+from bson.objectid import ObjectId
+from datetime import datetime
+import os, pymongo
+
+from todo_app.data.Item import Item
+
+def get_tasks(status=""):
+    if status == "":
+        tasks = get_database().find()
+    else:
+        tasks = get_database().find({"status": status})
+    
+    todo_tasks = []
+    for task in tasks:
+        readable_task = Item.from_mongodb(task)
+        todo_tasks.append(readable_task)
+    
+    return todo_tasks
+
+def get_task(id):
+    task_from_db = get_database().find_one({"_id": ObjectId(id)})
+    task = Item.from_mongodb(task_from_db)
+    return task
+
+def create_task(task_name, task_due_date="", task_notes=""):
+    task = task_layout(task_name, 'Not Started', task_due_date, task_notes)
+    added_task_data = get_database().insert_one(task)
+    return added_task_data.inserted_id
+
+def edit_task(task):
+    task_update = task_layout(task.name, task.status, task.due_date, task.notes)
+    get_database().find_one_and_update({'_id': ObjectId(task.id)}, {'$set': task_update})
+    updated_task = get_task(task.id)
+    return updated_task
+
+def delete_from_tasklist(id):
+    get_database().find_one_and_delete({"_id": ObjectId(id)})
+    return get_tasks()
```

Comment:
This return value isn't used, same for `edit_task` above. You can save yourself a round trip to the database by not calling `get_task(s)`